doc/hardware/peripherals/can/shell.rst

```diff
@@ -186,16 +186,20 @@ Received CAN frames matching the added filter(s) are printed to the shell. A few
 
 .. code-block:: console
 
-   # Flags  ID   Size  Data bytes
-   --       010   [8]  01 02 03 04 05 06 07 08
-   B-       010  [08]  01 02 03 04 05 06 07 08
-   BP       010  [03]  01 aa bb
-   --  00000010   [0]
-   --       010   [1]  20
-   --       010   [8]  remote transmission request
+   # Dev # Flags  ID   Size  Data bytes
```

Comment:
Did you mean to have that second `#` sign or should it be like this?

```suggestion
   # Dev Flags    ID   Size  Data bytes
```

Reply:
Ouch! No, the intention was, as you guessed, to only leave a hashmark at the beginning of the line. Fixed, thanks!